tests/test_addons.py

```diff
@@ -353,29 +355,29 @@ def setUpClass(cls):
         set_up_storage([DummyGuidFile], storage.MongoStorage)
 
 
-class TestAddonFileViewHelpers(OsfFileTestCase):
+# class TestAddonFileViewHelpers(OsfFileTestCase):
```

Comment:
If this is no longer necessary, let's remove it instead of commenting it out. Otherwise, please add a comment indicating why it was commented out.